### PR TITLE
Negative and positive  C-tagger in 81X

### PR DESCRIPTION
**Pull request by kovitang**
We added negative and positive C-tagger which will be use in BTV to identify the mistag rate.

@mverzett and @imarches are interested in following this

### CODE REVIEW

File: RecoBTag/CTagging/python/charmTaggerProducer_cfi.py [hunk 1]
```diff
@@ -32,3 +37,41 @@
    variables = c_vs_b_vars_vpset
    )
 
+#
+# Negative tagger
```
Comment (slava77) [inline RecoBTag/CTagging/python/charmTaggerProducer_cfi.py]: cfi files aren't really meant for definitions of multiple modules.
It should technically be just one and a cfi the file name better match the name of the module or PSet instance defined inside it.
Please move the various definitions to charmTaggerProducer_cff

File: RecoBTag/CTagging/python/positiveCharmTagJetTags_cfi.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+pfPositiveCombinedCvsLJetTags = cms.EDProducer(
```
Comment (slava77) [inline RecoBTag/CTagging/python/positiveCharmTagJetTags_cfi.py]: please match the file name closer to the module instance definitions.
Also, multiple module definitions better be done in cff files

**Comment by slava77** [comment]
@kovitang 
please confirm that you are working on the fixes mentioned during code review

**Comment by mverzett** [comment]
Hi, I can confirm for him, given the time zone difference :). We are testing the changes and they look good so far.

**Comment by kovitang** [comment]
@slava77 cfi now contains one module and file names are matched. Other modules are moved to cff. 
We also fixed one file for CTag DQM.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/14449	f6d59e6
- code changes are in line with the description: support for pos/neg c-taggers is added (config files and declarations to PAT tools); the modules are not running in the standard relval matrix workflows
- jenkins tests pass and comparisons with baseline show no difference

**Pull request merged**